service/ui_plugin_perfdash.go

```diff
@@ -42,7 +42,7 @@ func perfDashGetTasksForVersion(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	project, err := model.FindProject(v.Revision, projectRef)
+	project, err := model.FindProjectFromVersionID(v.Revision)
```

Comment:
Should this be v.Identifier?

Reply:
Good catch, thanks. I think it should be `Id` though. Fixed.